typing_extensions/src/typing_extensions.py

```diff
@@ -1123,6 +1125,26 @@ class Point2D(TypedDict):
         """
 
 
+if hasattr(typing, "is_typeddict"):
+    is_typeddict = typing.is_typeddict
+else:
+    def is_typeddict(tp):
+        """Check if an annotation is a TypedDict class
+        For example::
+            class Film(TypedDict):
+                title: str
+                year: int
+            is_typeddict(Film)  # => True
+            is_typeddict(Union[list, str])  # => False
+        """
+        typeddict_types = [_TypedDictMeta]
+        try:
+            typeddict_types.append(typing._TypedDictMeta)
+        except AttributeError:
+            pass
+        return isinstance(tp, tuple(typeddict_types))
```

Comment:
```suggestion
        TYPEDDICT_TYPES = (typing._TypedDictMeta, _TypedDictMeta) if hasattr(typing, "_TypedDictMeta") else (_TypedDictMeta,)


        def is_typeddict(tp):
         """Check if an annotation is a TypedDict class
         For example::
             class Film(TypedDict):
                 title: str
                 year: int
             is_typeddict(Film)  # => True
             is_typeddict(Union[list, str])  # => False
         """
         return isinstance(tp, TYPEDDICT_TYPES)
```
I don't see the point of not moving this into a constant.

Reply:
Totally happy to make this change. Waiting on @srittau to see if there are any other concerns.

Reply:
I agree with this suggestion, but please prefix TYPEDDICT_TYPES with an underscore to indicate that it's private (and don't use one-space indents :) ).